documentation/modules/con-standard-topic.adoc

```diff
@@ -27,3 +27,7 @@ So, for example:
 
 In the standard address space, the first level must always be a defined topic address; that is, `#` and `+` are not valid as the first characters of a subscribing address.
 
+== Known issue with creating a subscriber on a hierarchical topic
+
+A known issue exists where creating a subscriber on a hierarchical topic in {ProductName} causes the broker to instead create it as a competing consumer (handling the address like a queue rather than a topic). For more information about the specific workaround for your client, see the applicable client example section in link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#assembly-connecting-applications-messaging[].
```

Comment:
For a link you need to fill in text in the [] .. it's only xrefs that fill those in automatically.

Reply:
Done